documentation/proposals/Proposal - Rumble.md

```diff
@@ -0,0 +1,45 @@
+# Summary
+
+Proposal API for gamepad vibration/rumble control. 
```

Comment:
I think we should stick to one word, ie. `Proposal API for gamepad vibration control. `

Reply:
Fixed.